person.rb

```diff
@@ -0,0 +1,21 @@
+class Person
+  attr_accessor :name, :age
+  attr_reader :id
+
+  def initialize(age:, name: 'Unknown', parent_permission: true)
+    @id = id
```

Comment:
- [ ] **Required:** Kindly make some changes to how the id of a person is generated. Currently, the `@id` would throw an error because `id` is neither supplied in the initializer nor anywhere else. One way to fix is to use the [Ruby Random class.](https://ruby-doc.org/core-2.4.0/Random.html).

Consider:

`@id = Random.rand(1..1000)`

Reply:
ok thanks I will